poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.0 and should not be changed by hand.
```

Comment:
The previous lock file was generated with Poetry v1.8.5, but this update was generated with Poetry v1.8.0.

Could you update to the latest patch version of Poetry (v1.8.5) before running `poetry lock --no-update` to make sure we don't inadvertently introduce any dependency issues?